Aptoide/aptoide-client-v8
# [feature/ASV-1911] Redesign Earn AppCoins

**What does this PR do?**

   This PR redesigns Earn AppCoins Credits bundle. It also reorganizes the bundles into packages among other minor optimizations.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] BundlesAdapter.java
- [ ] Classes in package cm.aptoide.pt.home.bundles.appcoins

**How should this be manually tested?**

  Check that the designs matches the one in Zeplin, check that everything works (clicking in apps, see more and click on the card itself).

**What are the relevant tickets?**

  [ASV-1911](https://aptoide.atlassian.net/browse/ASV-1911)


**Code Review Checklist**
